store/src/java/com/zimbra/qa/unittest/TestServiceServlet.java

```diff
@@ -223,8 +223,8 @@ public void testDeployZimlet() throws Exception {
         String url = baseURL + "deployzimlet";
         PostMethod method = new PostMethod(url);
         addAuthTokenHeader(method, mbox.getAuthToken().getValue());
-        method.addRequestHeader(ZimletUtil.PARAM_ZIMLET, "com_zimbra_phone");
-        File zimletFile = new File("/opt/zimbra/zimlets/com_zimbra_phone.zip"); //standard zimlet, should always be there. No harm in redeploying it
+        method.addRequestHeader(ZimletUtil.PARAM_ZIMLET, "com_zimbra_unittest");
```

Comment:
Perhaps use a constant for this zimlet name?

Reply:
@fatkudu what do you mean?

Reply:
There are several references to "com_zimbra_unittest" in this file - suggesting using a constant for that, say "TEST_ZIMLET".  Not a big deal but that is what I mean.